### PR TITLE
Create DKG vector tests

### LINKED ISSUE #567: Create DKG vector tests

### PR DESCRIPTION
**Pull request by natalieesk**
This closes #567

### CODE REVIEW

File: frost-core/src/tests/vectors_dkg.rs [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+        VerifyingKey::deserialize(json_to_element::<C>(&inputs["verifying_key"])).unwrap();
+
+    PublicKeyPackage {
+        header: Header::default(),
+        verifying_shares,
+        verifying_key,
+    }
+}
+
+/// TODO documentation for test
```
Comment (conradoplg) [inline frost-core/src/tests/vectors_dkg.rs:187]: Do you still want to document this?

**Review by conradoplg — APPROVED** [review]
This is super useful, thank you!

There is just a lingering TODO, up to you if you want to do that now or later. I'll leave this approved so feel free to mark that comment as resolved or address it.

**Comment by conradoplg** [comment]
Ah, there is also a merge conflict

File: frost-core/src/tests/vectors_dkg.rs (hunk 1 above)
Comment (natalieesk) [reply frost-core/src/tests/vectors_dkg.rs]: Done in [b049dfa](https://github.com/ZcashFoundation/frost/pull/587/commits/b049dfa139cb74a73de927e5ca8d24a662639d38)

**Review by conradoplg — APPROVED** [review]

**Pull request merged**